### DISCUSSION TITLE
Using a background page

### DISCUSSION THREAD

**Discussion by vldslv-a** [remix-run/react-router#10723] [Show and tell / tips]
Hey, I'm upgrading to version 6.14.2 and I have a question how to use the background page when opening a modal window using createBrowserRouter? Could you provide an example or recommendation on how to do this?

It doesn't really work for me because the background page has changed.: https://github.com/remix-run/react-router/blob/dev/examples/modal/src/App.tsx#L56-L70

**Comment by maksis** [comment]
https://github.com/remix-run/react-router/discussions/9864
